recipes-test/images/secondary-image.bb

```diff
@@ -10,9 +10,10 @@ LICENSE = "MIT"
 IMAGE_INSTALL_remove = " \
                         aktualizr \
                         aktualizr-auto-prov \
+                        aktualizr-auto-prov-creds \
                         aktualizr-ca-implicit-prov \
+                        aktualizr-ca-implicit-prov-creds \
```

Comment:
Also `aktualizr-uboot-env-rollback`?

Reply:
You're probably right. Fixed.

Reply:
`oe-selftest -r updater.SecondaryTests` passed after the fix.